[SOAR-1896] Flagged Hash on Asset (#281)
rapid7/insightconnect-workflows: workflows/Quarantine_Asset_with_Insight_Agent_from_InsightIDR_UBA_Alert/help.md

# Description

> Quarantining a compromised asset can limit the scope of an attack and buy valuable time to investigate and contain the threat. This workflow quarantines an asset that has an Insight agent on it from an InsightIDR UBA alert.

Reviewer: ```suggestion
Quarantining a compromised asset can limit the scope of an attack and buy valuable time to investigate and contain the threat. This workflow triggers on an InsightIDR UBA alert to quarantine an asset with the Insight Agent.
```

This workflow can be used with the following types of UBA alerts:

`Brute Force - Local Account`

`Flagged Hash on Asset`

`Flagged Process on Asset`

# Key Features

* **Break the Kill Chain** - Quarantining an asset can quickly and effectively interrupt an attacker’s kill chain.
* **Buy Time to Investigate and Remediate** - In a pinch, quarantining an asset can limit your threat exposure and buy your team valuable time to investigate and contain a threat. 

# Requirements

* InsightIDR
* Insight Agent on the asset to be quarantined

# Documentation

## Setup

Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.

> Once the workflow has been imported, **It is recommended that you add you add the appropriate emails to the `Block Domain Account Decision` step so that the appropriate people are alerted that action needs to be taken**

Reviewer: ```suggestion
Once the workflow has been imported, **it is recommended that you add the appropriate email addresses to the `Block Domain Account Decision` step so that the appropriate users are alerted that action needs to be taken**
```

Activate the workflow in order to trigger it.

> Log into IDR and go to the automation tab. From there go to Alert Triggers. Click on Create Alert Trigger.

Reviewer: ```suggestion
Log into InsightIDR and go to the Automation tab, then Alert Triggers. Click on Create Alert Trigger.
```

Select Custom InsightConnect Workflows, then `Quarantine asset with Insight Agent from InsightIDR UBA Alert`,

> then chose they type of alert you would like to trigger on. For example: Flagged Hash on Asset.

Reviewer: ```suggestion
then choose the type of alert you would like to trigger on. For example: Flagged Hash on Asset.
```

This will set up the workflow to run any time the selected alert is triggered.

# Usage

> The workflow will run automatically when the selected InsightIDR UBA alert is created

Reviewer: ```suggestion
The workflow will run automatically when the selected InsightIDR UBA alert is created.
```
